app/views/shared/_navigation_bar.html.erb

```diff
@@ -1,21 +1,38 @@
 <nav class="navbar navbar-expand-lg navbar-dark bg-primary justify-content-between" data-turbolinks-animate-persist="true">
   <a class="navbar-brand" href="#"><i class="fas fa-sitemap"></i> Caucus</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggler" aria-controls="navbarToggler" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
   <div class="collapse navbar-collapse" id="navbarToggler" style="justify-content: right;">
-    <% if user_signed_in? && current_user.has_role?(:admin) %>
-    <div class="navbar-nav mr-4">
+    <% if user_signed_in? %>
+    <div class="navbar-nav mr-1">
+      <%= image_tag current_user.avatar.variant(resize_to_limit: [25, 25]) if current_user.avatar.attached? %>
```

Comment:
Oh! One additional change: can the profile image be moved from here to be right next to the email address (that spawns the dropdown menu, ~L25)? It seems like it belongs paired with that rather than just as the "first nav item".